getting-started/templates/Dependencies/Elasticsearch/README.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+SystemLink Enterprise uses a distributed search engine called [Elasticsearch](https://www.elastic.co/elasticsearch) to enable fast and scalable search capabilities.
+
+## Installing Elasticsearch
+
+NI provides a [Helm chart](https://needlink.com) that installs Elasticsearch on a Kubernetes cluster. This chart wraps a [Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/elasticsearch) from [Bitnami](https://bitnami.com/) to add features such as automatic user provisioning.
```

Comment:
@zraul123 I'm not sure this is correct, but I wanted to separate the different hyperlinks so that the user knows that there are two of them. Feel free to rearrange them

Reply:
@WillCawley This is correct, thank you! :)